modules/tinymce/src/core/test/ts/browser/focus/FocusControllerTest.ts

```diff
@@ -59,7 +59,14 @@ UnitTest.asynctest('browser.tinymce.core.focus.FocusControllerTest', function (s
     LegacyUnit.equal(FocusController.isEditorContentAreaElement(contentAreaElm2), true, 'Should be true since tox-edit-area__iframe is content area container element');
   });
 
-  TinyLoader.setup(function (editor, onSuccess, onFailure) {
+  suite.test('isUIElement on editor sibling is false', (editor) => {
```

Comment:
Would it be a possible to add a test to `DomUtilsTest.ts` that specifically tests `dom.getParent` and `dom.getParents` as well?